src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -172,25 +173,33 @@ public override Task StartDefaultOsBrowserAsync(string url)
                     throw new MsalClientException(MsalError.LinuxXdgOpen, MsalErrorMessage.LinuxOpenAsSudoNotSupported);
                 }
 
+                List<string> openTools = new List<string>();
+                if (isBrokerEnabled)
+                {
+                    openTools.Add("microsoft-edge");
+                }
+
+                openTools.Add("xdg-open");
+                openTools.Add("gnome-open");
+                openTools.Add("kfmclient");
+
                 try
                 {
                     ProcessStartInfo psi = null;
-                    foreach (string openTool in new[] { "xdg-open", "gnome-open", "kfmclient" })
-                    {
-                        if (TryGetExecutablePath(openTool, out string openToolPath))
-                        {
-                            psi = new ProcessStartInfo(openToolPath, url)
-                            {
-                                RedirectStandardOutput = true,
-                                RedirectStandardError = true
-                            };
-
-                            Process.Start(psi);
-
-                            break;
-                        }
-                    }
 
+                    if (psi == null)

```

Comment:
This check is not needed, it's always true.

Reply:
Missed to remove. Thanks